src/test/kotlin/no/nav/syfo/aktivitetskrav/api/AktivitetskravApiSpek.kt

```diff
@@ -399,7 +399,7 @@ class AktivitetskravApiSpek : Spek({
                     it("Returns historikk when aktivitetskrav has status NY_VURDERING") {
                         aktivitetskravRepository.createAktivitetskrav(
                             nyAktivitetskrav.copy(
-                                createdAt = nowUTC().minusWeeks(2) // createdAt same date as stoppunkt
+                                createdAt = nowUTC().minusWeeks(2).minusDays(1) // createdAt same date as stoppunkt
```

Comment:
Her ser man jo heller ikke hvorfor vi må gjøre minusDays(1), er det dumt? 🤔

Reply:
Blir litt magisk her ja. Kunne man gjort `createdAt =  OffsetDateTime.of(nyAktivitetskrav.stoppunktAt)` på et vis?